cmdutils/switches/switches.go

```diff
@@ -97,10 +133,22 @@ func (s *Switches) All() sets.String {
 	return names
 }
 
+// EnabledByDefault returns names of all enabled items
+func (s Switches) EnabledByDefault() sets.String {
+	names := make(sets.String)
```

Comment:
Saw this too late: It should be `sets.NewString` instead of 'custom' `make` calls.

Reply:
At least in version we use, there's no `NewString` method

Reply:
In newer versions this method is also no present: https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/sets/string.go